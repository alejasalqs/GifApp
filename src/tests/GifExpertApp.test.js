const { shallow } = require("enzyme");
const { default: GiftExperApp } = require("../GifExpertApp");
import React from 'react';

describe('Pruebas en el archivo GifExpertApp.js', () => {
    
    const wrapper = shallow( <GiftExperApp />)
    
    test('Debe renderizarse correctamente ', () => {
        expect(wrapper).toMatchSnapshot();
    });
})
