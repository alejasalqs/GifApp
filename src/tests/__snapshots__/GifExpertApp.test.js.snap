// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el archivo GifExpertApp.js Debe renderizarse correctamente  1`] = `
<Fragment>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <br />
  <ol>
    <GiftGrid
      category="Pokemon"
      key="Pokemon"
    />
  </ol>
</Fragment>
`;
